en/services/ftp.md

```diff
@@ -51,13 +51,14 @@ Byte Index | C version | Content | Value | Explanation
 The opcodes that may be sent by the GCS (client) to the drone (server) are listed below.
 
 <!--  uint8_t enum Opcode: https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_ftp.h -->
-     
+
+
 Opcode | Name | Description
 --- | --- | ---
 0   | None | Ignored, always ACKed
 <span id="TerminateSession"></span> 1 | TerminateSession | Terminates open Read `session`.<br>- Closes the file associated with (`session`) and frees the session ID for re-use. 
 <span id="ResetSessions"></span> 2   | ResetSessions | Terminates *all* open read sessions.<br>- Clears all state held by the drone (server); closes all open files, etc.<br>- Sends an ACK reply with no data. <!-- Note, is same as Terminate, but does not check if file session exists -->
-<span id="ListDirectory"></span> 3   | ListDirectory | List files and directories in `<path>` from `<offset>`.<br>- Opens the directory (`path`), seeks to (`offset`) and fills the result buffer with `NULL`-separated filenames (files also include tab-separated file size) and directory names. Sends an ACK packet with the result buffer on success, otherwise a NAK packet with an error code.<br>- The directory is closed after the operation, so this leaves no state on the server.
+<span id="ListDirectory"></span> 3   | [ListDirectory](#list_directory) | List directory contents in `<path>`, starting from file/directory at `<offset>`.<br>- Sends an ACK packet with the result buffer on success, otherwise a NAK packet with an error code.<br>- The directory is closed after the operation, so this leaves no state on the server.
```

Comment:
FYI This now links down. Have just done it for this one, but would roll down link out more widely after this merges. I didn't update the full description at this point because that is captured below (ie no need for duplication, this can be high level).